File: app/src/main/java/com/roman/batsu/utils/BaseActivity.java
package com.roman.batsu.utils;

import android.annotation.SuppressLint;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.roman.batsu.R;

//создано для сплэш скрина

@SuppressLint("Registered")
public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        setTheme(R.style.AppTheme);
        super.onCreate(savedInstanceState);
    }
}

File: app/src/main/java/com/roman/batsu/utils/BaseActivity.kt
package com.roman.batsu.utils

import android.annotation.SuppressLint
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.roman.batsu.R

//создано для сплэш скрина

@SuppressLint("Registered")
class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(R.style.AppTheme)
        super.onCreate(savedInstanceState)
    }
}